tests/testthat/test-render.R

```diff
@@ -176,9 +176,16 @@ testthat::describe("render() renders output based on metadata$output field:", {
     metadata(teal_card(tr)) <- list(output = "md_document")
     teal.reporter::render(tr, quiet = TRUE)
     lines <- base::readLines("report.md", warn = FALSE)
-    testthat::expect_identical(
-      lines,
-      c(
+
+    # Pandoc >= 2.11.2 uses ATX headers (# header), older versions use Setext (header\n===)
+    pandoc_ver <- tryCatch(
+      utils::compareVersion(rmarkdown::pandoc_version(), "2.11.2") >= 0,
+      error = function(e) FALSE
+    )
+
+    if (pandoc_ver) {
```

Comment:
It seems like this function does the same check

<img width="804" height="422" alt="Image" src="https://github.com/user-attachments/assets/d6e0d812-fe95-4356-9458-49db2ff77348" />

```suggestion
    if (rmarkdown::pandoc_available("2.11.2")) {
```